front/Theme.tsx

```diff
@@ -4,9 +4,15 @@ import { useEffect } from 'react';
 
 const ThemeProvider = ({ children }: { children: JSX.Element }) => {
 	const colorScheme = useColorScheme();
+    const config = {
+        dependencies: {
+            "linear-gradient": require("expo-linear-gradient").LinearGradient,
```

Comment:
Please, use a proper import statement. Just tested it, it works :]

Reply:
wdym ?

Reply:
The documentation shows how we could do dynamic imports. This is not what we look for here.